src/content/developers/docs/consensus-mechanisms/pow/index.md

```diff
@@ -20,8 +20,17 @@ Proof-of-work is the mechanism that allows the decentralized Ethereum network to
 
 ## Proof-of-work and mining {#pow-and-mining}
 
+Decentralized consensus requires two things:
+
+- A common objective: extending the chain (motivated by economic incentives)
```

Comment:
Same as the other section, I think we already cover this adequately on this page or in the prerequsities for this page.